contracts/examples/guards/DelegateCallTransactionGuard.sol

```diff
@@ -46,4 +46,20 @@ contract DelegateCallTransactionGuard is BaseGuard {
     }
 
     function checkAfterExecution(bytes32, bool) external view override {}
+
+    /**
+     * @notice Called by the Safe contract before a transaction is executed via a module.
+     * @param to Destination address of Safe transaction.
+     * @param value Ether value of Safe transaction.
+     * @param data Data payload of Safe transaction.
+     * @param operation Operation type of Safe transaction.
+     * @param msgSender Account executing the transaction.
+     */
+    function checkModuleTransaction(
+        address to,
+        uint256 value,
+        bytes memory data,
+        Enum.Operation operation,
+        address msgSender
+    ) external override {}
```

Comment:
Should we add the delegate call check here too?

Reply:
I actually did, but it got lost while jumping between branches with different implementations, good catch